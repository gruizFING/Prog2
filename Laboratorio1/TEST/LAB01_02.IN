sumar 4 3
imprimir datos
conectar terminal servidor
pausa
sumar 1 2 3 4
salir