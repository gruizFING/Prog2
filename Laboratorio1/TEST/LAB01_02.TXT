sumar 4 3
Comando leido: sumar
Con los siguientes parametros: 4,3
imprimir datos
Comando leido: imprimir
Con los siguientes parametros: datos
conectar terminal servidor
Comando leido: conectar
Con los siguientes parametros: terminal,servidor
pausa
Comando leido: pausa
Sin parametros
sumar 1 2 3 4
Comando leido: sumar
Con los siguientes parametros: 1,2,3,4
salir
Fin!