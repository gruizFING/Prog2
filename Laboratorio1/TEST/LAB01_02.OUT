Comando leido: sumar
Con los siguientes parametros: 4,3
Comando leido: imprimir
Con los siguientes parametros: datos
Comando leido: conectar
Con los siguientes parametros: terminal,servidor
Comando leido: pausa
Sin parametros
Comando leido: sumar
Con los siguientes parametros: 1,2,3,4
Fin!